pyth-rng/config.yaml

```diff
@@ -0,0 +1,4 @@
+chains:
```

Comment:
At the risk of bikeshedding, I'm not the biggest fan of config files and prefer CLI interfaces like repeated args, I'm open to anything here but did we ever come to a conclusion on what configuration files among Pyth projects should be? Is it to be YAML everywhere? Just want to double check before we end up with multiple configuration styles.

Reply:
Yeah so I went back and forth on this config file. For this particular use case though, we're going to end up with a big list of different ethereum chains with corresponding RPC urls, and it seemed like this would end up being quite awkward to pass on the command line. I've kept everything else on the command line for now though.

We use YAML in most places for config (except typescript which uses .env sometimes, but that really sucks).